tests/suites/test_suite_psa_crypto_driver_wrappers.function

```diff
@@ -1738,3 +1738,92 @@ exit:
     mbedtls_test_driver_hash_hooks = mbedtls_test_driver_hash_hooks_init();
 }
 /* END_CASE */
+
+/* BEGIN_CASE */
+void copy_key_various( int source_usage_arg,
```

Comment:
As for key import I would prefer to:
1) Leverage test_suite_psa_crypto copy key tests to also test the copy of keys within the opaque driver.
2) Here in test_suite_psa_crypto_driver_wrappers, to just test the dispatch and testing the case where the driver returns in error: see hash_compute for an example.
By the way it seems that point 2) is not done for key import.